site/pages/retirements/[beneficiary]/index.tsx

```diff
@@ -42,6 +42,16 @@ export const getStaticProps: GetStaticProps<PageProps, Params> = async (
       throw new Error("No params found");
     }
 
+    // enforces lowercase urls
+    if (params.beneficiary !== params.beneficiary.toLowerCase()) {
```

Comment:
id move this below the nameservice redirect logic, just to avoid a potential double redirect

Reply:
Good one!

Reply:
Done!